package.json

```diff
@@ -25,9 +25,13 @@
     "@tanstack/react-query": "^5.28.8",
     "@tanstack/react-query-devtools": "^5.28.8",
     "@typescript-eslint/eslint-plugin": "^7.5.0",
+    "bignumber.js": "^9.1.2",
+    "bindings-js": "file:./src/temp/stellar-xdr-web",
```

Comment:
Temporarily including this locally in the repo. When the package is published, we'll clean everything up.